tests/integration_tests/scheduler/test_generic_driver.py

```diff
@@ -110,15 +111,16 @@ async def finished(iens, returncode):
     assert aborted_called
 
 
+@pytest.mark.flaky(reruns=5)
 async def test_kill_actually_kills(driver: Driver, tmp_path, pytestconfig):
     os.chdir(tmp_path)
 
     if (isinstance(driver, LsfDriver) and pytestconfig.getoption("lsf")) or (
         isinstance(driver, OpenPBSDriver) and pytestconfig.getoption("openpbs")
     ):
         # Allow more time when tested on a real compute cluster to avoid false positives.
-        job_kill_window = 10
-        test_grace_time = 20
+        job_kill_window = 60
```

Comment:
Has the need for this incrase been observed?

Reply:
I have not observed the flakiness since the creation of this ticket. Maybe it got scared off.

Reply:
We discussed that this could be a possible remedy for the flakiness we saw.

Reply:
Could be correct _only if_ it is observed in the second part of the integration test where --openpbs or --lsf is used.

Reply:
^it was apparently observed on bleeding, and on every build currently.